python/lsst/qserv/tests/dbLoader.py

```diff
@@ -51,10 +52,19 @@ def __init__(self, config, data_reader, db_name, multi_node, out_dirname):
 
         if self._multi_node:
             self.qAdmin = qservAdmin.QservAdmin('localhost:' + str(self.config['zookeeper']['port']))
-            self.nAdmin = nodeAdmin.NodeAdmin('worker1',
-                                              self.qAdmin,
-                                              wmgrSecretFile=self.config['wmgr']['secret'])
-            self.wmgr = self.nAdmin.wmgrClient()
+            self.nMgmt = nodeMgmt.NodeMgmt(self.qAdmin, wmgrSecretFile=self.config['wmgr']['secret'])
+
+            self.nodeNames = []
+            self.nAdmins = {}
+            self.nWmgrs = {}
+
+            for node in self.nMgmt.selectDict(nodeType='worker', state='ACTIVE'):
+                self.logger.info("Running on node: %s", node)
+                self.nodeNames.append(node)
+                self.nAdmins[node] = nodeAdmin.NodeAdmin(node,
+                                                         self.qAdmin,
+                                                         wmgrSecretFile=self.config['wmgr']['secret'])
+                self.nWmgrs[node] = self.nAdmins[node].wmgrClient()
```

Comment:
I think this could be simplified a little bit:
- using `select()` instead of `selectDict()` would save some typing
- `self.nAdmins` is not used below, I don't think you have to keep it
- `self.nodeNames` is also not needed, node names can be retrieved from the keys of `self.nWmgrs` dict (that is ignoring ordering for now)
So I'd rewrite it as::
```python
    self.nWmgrs = {}
    for node in self.nMgmt.select(nodeType='worker', state='ACTIVE'):
        self.logger.info("Running on node: %s", node.name())
        self.nWmgrs[node.name()] = node.wmgrClient()
```
And see changes below